modules/etcd-increase-db.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="etcd-increase-db_{context}"]
+= Increasing the database size for etcd
+
+You can set the disk quota in gigabytes for each etcd instance. If you set a disk quota for your etcd instance, the instance can increase up to the value that you specify.
+
+You can specify integer values from 8 to 32. The default value is 8. You can specify only increasing values.
+
+:FeatureName: Increasing the database size for etcd
+include::snippets/technology-preview.adoc[]
+
+[id="etcd-change-db-size_{context}"]
+== Changing the etcd database size
+
+To change the database size for etcd, complete the following steps.
```

Comment:
Is there a specific reason why users want to increase the size of the database, for example, to store the increasing workload/application data?

Up to you though!

Reply:
Good question!
@dusk125 What do you think of adding a line or two to explain why users might want to increase the size of their database? Or would that be obvious to most people who are reading these docs?

Reply:
Yeah I think that's a good idea. Pretty much the only reasons would be
1. If they've experienced a ["Low Space" alert](https://github.com/openshift/cluster-etcd-operator/blob/master/manifests/0000_90_etcd-operator_03_prometheusrule.yaml#L90)

This alert is an indicator that they're cluster is getting too big to fit in etcd (assuming automatic-defrag is working properly) and they could benefit from this feature immediately; once etcd runs out of space, writes will start failing.

2. If they're experiencing ["Excessive Database Growth" alert (warning)](https://github.com/openshift/cluster-etcd-operator/blob/master/manifests/0000_90_etcd-operator_03_prometheusrule.yaml#L100)

This is an indicator that the database *may* get too big (but hasn't yet) and they may benefit from the feature and avoid the write-failing issue. This is a warning because it's just going off of the previous growth; if the growth stops, then they don't need this feature.. but if it does, following this warning could give them time to increase the size before running into write-failures.

Other than those reasons, I would suggest people not mess with it :D